codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -161,6 +161,17 @@ public void onHttpServerUpgrade(Http2Settings settings) throws Http2Exception {
         connection().remote().createStream(HTTP_UPGRADE_STREAM_ID, true);
     }
 
+    @Override
+    public void flush(ChannelHandlerContext ctx) throws Http2Exception {
+        // Trigger pending writes in the remote flow controller.
+        connection().remote().flowController().writePendingBytes();
```

Comment:
The old code doesn't have this ... why do we need this if all we're doing is coalescing data frames?